scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -738,8 +774,18 @@ def __init__(self, config, logger, channels):
 
         self._placeholder_render = config.get("placeholder_render")
         if self._placeholder_render:
-            self._dll_cache = Cache(size=10)
-            self._param_cache = Cache(size=1000)
+            self._dll_cache = Cache(
+                size=config.get("dll_handle_cache_size"),
+                ttl=config.get("dll_handle_cache_ttl"),
+            )
+            self._param_cache = Cache(
+                size=config.get("placeholder_param_cache_size"),
+                ttl=config.get("placeholder_param_cache_ttl"),
+            )
+
+            self._cache_report_interval = 3600 * 12
```

Comment:
Since we already provide ability to fine-tune the cache, maybe it would be also helpful to configure the interval?

Reply:
You're probably right, but I'll hold off for now